backend_restful/education/tasks/join_institute.py

```diff
@@ -23,31 +23,52 @@ def __init__(self, data={}):
             education = db.education
             user_id = data.get("user_info")
             public_pages = db.public_pages
-            #query if already exist
-            ppageid = public_pages.find_one({"_id": ObjectId(public_page_id)})
 
+            # Inserting a dummy public_page object for testing
+            if(1):
+                data = {"_id": ObjectId(), "page_title":"Begum Rokeya university", "description": "BRUR Description", "designation":{
+                    {"_id": ObjectId(), "title":"parents", "tasks":
+                    {{
+                        {"_id": ObjectId()}
+                        {"title":"task11_name"}
+                    }
+                    {
+                        {"_id": ObjectId()}
+                        {"title":"task12_name"}
+                    }}}
+                    {"_id":ObjectId(), "title":"teacher","tasks":
+                    {{
+                        {"_id": ObjectId()}
+                        {"title":"task21_name"}
+                    }
+                    {
+                        {"_id": ObjectId()}
+                        {"title":"task22_name"}
+                    }}}}
+                
```

Comment:
here you need to perform the insert_one code and get the id of the inserted document